src/migrate/stub/ts.stub

```diff
@@ -2,20 +2,20 @@ import * as Knex from "knex";
 
 <% if (d.tableName) { %>
 export async function up(knex: Knex): Promise<Knex.SchemaBuilder> {
-    return knex.schema.createTable("<%= d.tableName %>", function (t: Knex.AlterTableBuilder) {
+    return knex.schema.createTable("<%= d.tableName %>", (t: Knex.AlterTableBuilder) => {
         t.increments();
         t.timestamps();
     });
-};
+}
```

Comment:
Why remove semicolons, though?

Reply:
Normal function declaration shouldn't have semicolon there. With arrow function, it would be obligatory though.

Reply:
@felixmosh So if I understand correctly, semicolons should be restored, since it's an arrow function?..

Reply:
@kibertoad , no, `up` is not an arrow function :]
The function that we pass to `createTable` is an arrow func, and there are semi-colons there.

Reply:
Fair enough.